sdk device brand data (#6226)
braze-inc/braze-docs: _docs/_user_guide/data_and_analytics/user_data_collection/sdk_data_collection.md

| Device IDFA      | iOS                | Device identifier for advertisers                     | Optionally used to track data. |
| Device locale    | Android, iOS       | The default locale of the device                      | This attribute is used to translate messages to a user's preferred language. |
| Device model     | Android, iOS       | The specific hardware of the device                   | This attribute is used to ensure messages are only sent to compatible devices. It can also be used within segmentation. |

> | Device brand     | Android       | The brand of device (i.e. Samsung)                   | This attribute is used to ensure messages are only sent to compatible devices. |

Reviewer: ```suggestion
| Device brand     | Android       | The brand of device (for example, Samsung)                   | This attribute is used to ensure messages are only sent to compatible devices. |
```
